conda_forge_tick/migrators/version.py

```diff
@@ -489,6 +489,15 @@ def pr_body(self, feedstock_ctx: FeedstockContext) -> str:
             and muid.get("PR", {}).get("state", None) == "open"
         ]
 
+        # Display the url so that the maintainer can quickly click on it
+        # in the PR body.
+        about = feedstock_ctx.attrs.get("about")
```

Comment:
is this guaranteed to exist? I'm seeing in the logs for awscli:
```
2020-03-28 17:49:09,443 ERROR    conda_forge_tick.auto_tick || NON GITHUB ERROR
Traceback (most recent call last):
  File "/root/repo/cf-scripts/conda_forge_tick/auto_tick.py", line 922, in main
    hash_type=attrs.get("hash_type", "sha256"),
  File "/root/repo/cf-scripts/conda_forge_tick/auto_tick.py", line 224, in run
    body=migrator.pr_body(feedstock_ctx),
  File "/root/repo/cf-scripts/conda_forge_tick/migrators/version.py", line 495, in pr_body
    upstream_url = about.get("dev_url", "") or about.get("home", "")
AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
I just pushed a fix for this one.